x-pack/legacy/plugins/ml/public/jobs/new_job/wizard/steps/job_type/job_type.html

```diff
@@ -1,274 +0,0 @@
-                  />
-                  <path d="M7 6h1v5H7zM7 12h1v1H7z" />
-                </g>
-              </svg>
-              <span class="euiCallOutHeader__title">{{indexWarningTitle}}</span>
-            </div>
-            <div class="euiText euiText--small">
-              <p>
-                <span
-                  i18n-id="xpack.ml.newJob.wizard.jobType.howToRunAnomalyDetectionDescription"
-                  i18n-default-message="Anomaly detection can only be run over indices which are time based."
-                ></span>
-                <br/>
-                <a
-                  class="euiLink euiLink--primary"
-                  href="#jobs/new_job\"
-                  i18n-id="xpack.ml.newJob.wizard.jobType.selectDifferentIndexLinkText"
-                  i18n-default-message="Select a different index"
-                ></a>
-              </p>
-            </div>
-          </div>
-        </div>
-        <div class="euiSpacer euiSpacer--xxl"></div>
-      </div>
-
-      <div ng-show="recognizerResults.count!==0" class="recognizer-section">
-        <div class="euiText">
-          <h3
-            class="euiTitle euiTitle--small"
-            i18n-id="xpack.ml.newJob.wizard.jobType.useSuppliedConfigurationTitle"
-            i18n-default-message="Use a supplied configuration"
-            >
-          </h3>
-          <p
-            i18n-id="xpack.ml.newJob.wizard.jobType.useSuppliedConfigurationDescription"
-            i18n-default-message="The fields in your data have been recognized as matching known configurations.
-            Select to create a set of machine learning jobs and associated dashboards."
-          ></p>
-        </div>
-        <div class="euiSpacer euiSpacer--m"></div>
-        <ml-data-recognizer
-          class-name='"euiFlexGrid euiFlexGrid--gutterLarge euiFlexGrid--fourths"'
-          item-class-name='""'
-          index-pattern='indexPattern'
-          saved-search='savedSearch'
-          results=recognizerResults/>
-        <div class="euiSpacer euiSpacer--xxl"></div>
-      </div>
-
-      <div class="euiText">
-        <h3
-          class="euiTitle euiTitle--small"
-          i18n-id="xpack.ml.newJob.wizard.jobType.useWizardTitle"
-          i18n-default-message="Use a wizard"
-        ></h3>
-        <p
-          i18n-id="xpack.ml.newJob.wizard.jobType.useWizardDescription"
-          i18n-default-message="Use one of the wizards to create a machine learning job to find anomalies in your data."
-        ></p>
-      </div>
-
-      <div class="euiSpacer euiSpacer--m"></div>
-
-      <div class="euiFlexGrid euiFlexGrid--gutterLarge euiFlexGrid--fourths">
-        <div class="euiFlexItem" ng-class='{disabled: isTimeBasedIndex===false}'>
-          <a ng-href="{{getUrl('#jobs/new_job/single_metric')}}" ng-class='{disabled: isTimeBasedIndex===false}' class="euiLink synopsis" data-test-subj="mlJobTypeLinkSingleMetricJob">
```

Comment:
The `data-test-subj="mlJobTypeLinkSingleMetricJob"` for this card (and also for the other cards) seems to be lost with the current changes (whjich also makes the x-pack ciGroup3 tests fail). The new cards should have the same `data-test-subj` attributes attached.